<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="bRender">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>460</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Render</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Set Camera Parameters</string>
    </property>
    <widget class="QSlider" name="hsD">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>5</number>
     </property>
     <property name="maximum">
      <number>25</number>
     </property>
     <property name="value">
      <number>15</number>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="hsFOV">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>261</width>
       <height>18</height>
      </rect>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>120</number>
     </property>
     <property name="value">
      <number>80</number>
     </property>
     <property name="tracking">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lbD">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Distance from Black Hole</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbFOV">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Field of View (FOV)</string>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>551</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbRenderProgress">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>460</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>301</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>View Single Star</string>
    </property>
    <widget class="QSlider" name="hsXoff">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-20</number>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TickPosition::TicksAbove</enum>
     </property>
    </widget>
    <widget class="QSlider" name="hsYoff">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-20</number>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="tracking">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TickPosition::TicksAbove</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lbXoff">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>x deviation</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbYoff">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>y deviation</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
